Formula/python-internal.rb

```diff
@@ -0,0 +1,350 @@
+class PythonInternal < Formula
```

Comment:
Not a strong opinion, but mayhaps we call this `python-bootstrap`? Since that's what we plan to use it for...

Some of the changes suggested in https://github.com/Homebrew/homebrew-core/pull/68528 might also be useful here.

Reply:
Once #68528 is merged we can sync both formulae. They might diverge slightly over time but I'll keep an eye on this from time to time, and as long as we have a working bottle for this it's not that important to have them 100% aligned all the time.